src/central-server/admin-service/core-api/src/main/java/org/niis/xroad/cs/admin/api/domain/ManagementRequestView.java

```diff
@@ -49,11 +50,14 @@ public class ManagementRequestView {
     private String securityServerOwnerName;
```

Comment:
This DTO should fully replace ManagementRequestInfoDto, this one was created during refactoring, but never fully utilized